addons/survey/static/src/question_page/description_page_field.xml

```diff
@@ -12,6 +12,11 @@
                 </div>
             </t>
         </xpath>
+        <xpath expr="//span[@t-esc='formattedValue']" position="after">
```

Comment:
As said on the task, not a fix, so should not target 16.0.

Is also modified templates and could break customization.
16.0 is getting close to its first saas release, we should stick to our stable policy now :)

Could be split in a small master PR though.